triplea-game/triplea
# Use nested test fixtures to reduce duplication in test method names

## Overview

Numerous test methods duplicate the name of the method/feature under test as a prefix on the method name.  This leads to unusually long method names in some cases.  This PR merges such related test methods under a single `@Nested` test fixture named after the method/feature under test in order to reduce the duplication.

These changes also fix several "method can be static" warnings flagged by static analysis.

## Functional Changes

None.  **This PR only contains changes to test code.**

## Refactoring Changes

For tests that were modified, I reduced the accessibility of all public members to package-private, as JUnit 5 permits.

## Manual Testing Performed

Verified the same number of unit tests are run on this branch as on `master` (with one exception; see below).

## Additional Review Notes

Please review with whitespace changes ignored, as the vast majority of changes are due to indentation.
